src/puptoo/process/profile.py

```diff
@@ -624,8 +630,10 @@ def system_profile(
             profile["releasever"] = profile["yum_updates"].get("releasever")
             profile["basearch"] = profile["yum_updates"].get("basearch")
 
-    if pmlog_summary or ros_config:
+    if (pmlog_summary_pcp_zeroconf or ros_collect or    # ros new collection
+            pmlog_summary or ros_config):               # ros old collection
         profile["is_ros"] = True
+        profile["is_pcp_raw_data_collected"] = bool(pcp_raw_data)
```

Comment:
Using this field, you need to add it under `platform_metadata` as well [here](https://github.com/RedHatInsights/insights-puptoo/blob/1469fc638441724f3444971f7616f626ea388330/src/puptoo/app.py#L246-L247)

Reply:
@r14chandra, thanks for the review.    

Updated the code to [send "is_pcp_raw_data_collected" under platform_metadata](https://github.com/RedHatInsights/insights-puptoo/pull/329/commits/41707f2d164fcbd0b2be373d1573372b3111bb57)